core/src/test/java/io/questdb/test/griffin/AsOfJoinFuzzTest.java

```diff
@@ -112,6 +128,10 @@ private void assertResultSetsMatch0(JoinType joinType, boolean exerciseIntervals
             default:
                 throw new IllegalArgumentException("Unexpected join type: " + joinType);
         }
+        if (maxTolerance != -1) {
+            long toleranceSeconds = rnd.nextLong(maxTolerance) + 1;
+            onSuffix += " tolerance " + toleranceSeconds + "s ";
```

Comment:
nit: would be great to assert that the right-hand timestamps are within the tolerance interval in the result sets.

Reply:
excellent idea!